README.adoc

```diff
@@ -88,13 +88,13 @@ version: 5.0
 
 . Tag the latest commit in your release branch with a `v` followed by the version number.
 
-. If you have made any changes since creating your release branch, merge those changes back into the `.z` branch.
+. Create a `\{version\}.z` branch from release branch. It ensures that all changes made since creating your release branch will be applied to the new `.z` branch. For example `5.0.z` branch.
```

Comment:
So we create a `.z` branch to allow others to add content to future path releases of that minor version? Should we explain that here?

Reply:
Basically we create `.z` branch in the time when given Major/Minor release if tagged (i.e. it the time it's released). Generally `.z` branches are used for some smaller fixes of given branches. I'm not sure how we should explain it there, feel free to suggest something.